### PR TITLE
[10.0][IMP] analytic_location

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Related to: https://www.eficent.com/web#id=987&view_type=form&model=project.task

### CODE REVIEW

File: analytic_location/model/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _default_analytic_account_id(self):
+        picking_type_id = self.picking_id._context.get(
+            'default_picking_type_id')
+        analytic_account = self.env['account.analytic.account'].search(
+            ['|', ('picking_type_id', '=', picking_type_id),
+             ('picking_type_out_id', '=', picking_type_id),
+             ('parent_id', '=', 1)])
```
Comment (LoisRForgeFlow) [inline analytic_location/model/stock_move.py:18]: why this?

Comment (AaronHForgeFlow) [reply analytic_location/model/stock_move.py:18]: This is necessary only when in the company all projects are children of a master project, father of all. But as this is a customization better to move to a  custom repo.

File: analytic_location/model/stock_move.py [hunk 2]
```diff
@@ -14,8 +14,8 @@ def _default_analytic_account_id(self):
             ['|', ('picking_type_id', '=', picking_type_id),
              ('picking_type_out_id', '=', picking_type_id),
              ('parent_id', '=', 1)])
-        if analytic_account:
-            return analytic_account.id
+        if len(analytic_account):
```
Comment (MiquelRForgeFlow) [inline analytic_location/model/stock_move.py:19]: wrong commit. just put limit=1 in the search above instead.

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by AaronHForgeFlow** [comment]
Superseded by https://github.com/ForgeFlow/eficent-odoo-addons/pull/232
Reason: incorrect target branch

**Comment by AaronHForgeFlow** [comment]
Plz don't delete branch, will delete later

**Pull request closed without merging**